CMakeLists.txt

```diff
@@ -1,32 +1,79 @@
 cmake_minimum_required(VERSION 3.1)
 
-
-set(CMAKE_DISABLE_SOURCE_CHANGES ON)
-set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
```

Comment:
Why remove these? It's a good idea to prevent builds in the source dir

Reply:
Good point.